### PR TITLE
systemd: ignore comments in /etc/crypto-policies/config

### PR DESCRIPTION
**Pull request by tomasmatus**
fixes: #21311

### CODE REVIEW

File: pkg/systemd/overview-cards/cryptoPolicies.jsx [hunk 1]
```diff
@@ -61,7 +61,8 @@ export const CryptoPolicyRow = () => {
                     } catch (error) {
                         console.warn("Failed to get current crypto policy:", error.toString(),
                                      "; falling back to /etc/crypto-policies/config");
-                        setCurrentCryptoPolicy(contents.trim());
+                        const filteredContent = contents?.split('\n').filter(line => !line.startsWith("#")).join('\n');
```
Comment (martinpitt) [inline pkg/systemd/overview-cards/cryptoPolicies.jsx:64]: contents must exist at this point, but it doesn't hurt.

**Review by martinpitt — APPROVED** [review]
Whoops! So apparently `update-crypto-policy` does not write these files with comments. Thanks for fixing!

(Test case would be nice, though)

File: pkg/systemd/overview-cards/cryptoPolicies.jsx (hunk 1 above)
Comment (tomasmatus) [reply pkg/systemd/overview-cards/cryptoPolicies.jsx:64]: whoop, that was my autocomplete who happened to be confused by the lack of TS in a JS file

File: pkg/systemd/overview-cards/cryptoPolicies.jsx [hunk 2]
```diff
@@ -61,7 +61,8 @@ export const CryptoPolicyRow = () => {
...
+                        setCurrentCryptoPolicy(filteredContent?.trim() ?? null);
```
Comment (cockpituous) [inline pkg/systemd/overview-cards/cryptoPolicies.jsx:65]: These 2 added lines are not executed by any test.

**Pull request merged**